web-client/src/client_configuration.rs

```diff
@@ -0,0 +1,129 @@
+use std::str::FromStr;
+
+use tsify::Tsify;
+use wasm_bindgen::prelude::{wasm_bindgen, JsError, JsValue};
+
+use nimiq_primitives::networks::NetworkId;
+
+/// Use this to provide initialization-time configuration to the Client.
+/// This is a simplified version of the configuration that is used for regular nodes,
+/// since not all configuration knobs are available when running inside a browser.
+#[wasm_bindgen]
+pub struct ClientConfiguration {
+    #[wasm_bindgen(skip)]
+    pub network_id: NetworkId,
+    #[wasm_bindgen(skip)]
+    pub seed_nodes: Vec<String>,
+    #[wasm_bindgen(skip)]
+    pub log_level: String,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Tsify)]
+#[serde(rename_all = "camelCase")]
+pub struct PlainClientConfiguration {
```

Comment:
The `struct` and their members are missing `rustdoc`

Reply:
Again, self-explanatory.